ui/lobby/src/view/realTime/chart.ts

```diff
@@ -31,8 +31,10 @@ const clockX = (dur: number) => {
 };
 
 function renderPlot(ctrl: LobbyController, hook: Hook) {
-  const bottom = Math.max(0, ratingY(hook.rating) - 2),
-    left = Math.max(0, clockX(hook.t) - 2),
+  const chart = document.querySelector('.hooks__chart') as HTMLElement;
+  const fontSize = parseFloat(window.getComputedStyle(chart).fontSize);
```

Comment:
this `renderPlot` function is called dozens of times every second. These functions producing VDOM are supposed to be extremely cheap.
It's not appropriate to use `document.querySelector` and even less so `getComputedStyle` here.